docs/baseline-protocol-code/branch-naming.md

```diff
@@ -0,0 +1,26 @@
+# Branch Naming Standard
+
+## Main
+
+*main* - main development branch, feature, bugfix and release branches branched from it. Changes only through the PR process.
+
+## Feature
+
+*feature/XYZ-this-is-a-new-feature-branch* - related issue with number id XYZ.
```

Comment:
@ognjenkurtic what if a PR bundles multiple issues? How would that work? What is your suggestion?

Reply:
Good point, since there might be multiple issues opened for same thing.
We can also link the issues with PR, on the right side there is a section:
```
Development
Successfully merging this pull request may close these issues.
```

Maybe with that we won't have the need to also embed issue id in branch name, or maybe just one issue in branch name, and link all in PR this way? Or we could go with feature/X-Y-Z... for multiple issues X, Y and Z, since I doubt there will be more than couple of issues fixed with same PR? 
For blips since they are not part of this repo and can not be linked with github features, we should put them in branch name, and there should not be situation with multiple blips handled i think.

Reply:
I believe it is enough that the branch name reflects at least one of them - the main one. The remaining issues can be referenced in the PR itself and closed accordingly. This is a rather rare case, i do not think we need to cover it with docs at this point. Let's keep things lean - if we see that this starts to happen more often, we can amend the standard.

Reply:
@ognjenkurtic can you please, add the requirement that each PR must reference all issues and BLIPs the PR addresses, either whole or partially?

Reply:
Sure thing, added.